readme.md

```diff
@@ -86,12 +98,12 @@ You can also access the Application Link details page by going to **Jenkins** >
 
 #### Part 2 - Create an Application Link to Jenkins
 
-The second part is done in Bitbucket Server and involves creating an Application Link to Jenkins. Many of the details you need to do this are on the Application Link details page mentioned in step 1. 
+The second part is done in Bitbucket Server and involves creating an Application Link to Jenkins. Many of the details you need to do this are on the Application Link details page mentioned in Part 1. 
```

Comment:
Since we're removing 'part' from the titles this can be lower case. 

```suggestion
The second part is done in Bitbucket Server and involves creating an Application Link to Jenkins. Many of the details you need to do this are on the Application Link details page mentioned in part 1. 
```